### ISSUE TITLE
Integrate TM Terminal Eclipse plugin

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#211]
In old `oasp4j-ide` we provided eclipse as software binary bundled with various plugins. One of them was TM Terminal:
https://marketplace.eclipse.org/content/tm-terminal

Now we download the software like eclipse. For plugins we can automatically install via Update-Site URL. However, for TM Terminal I could not find a regular update site. It only seems to be available in Marketplace and I can not find a way to automatically install it this way.

Options:
* figure out an update site for TM Terminal and integrate in `eclipse` commandlet (easiest solution).
* find a way to install marketplace items automatically (maybe via OOMPH).

**Comment by hohwille** [comment]
https://download.eclipse.org/tm/updates/4.5.100-SNAPSHOT/repository/

**Comment by hohwille** [comment]
Almost nothing is ever easy (esp. in this devon-ide):
```
devon eclipse add-plugin terminal
Installing eclipse plugin org.eclipse.tm.terminal.feature from http://download.eclipse.org/tm/updates/4.5.100-SNAPSHOT/repository/
org.eclipse.m2e.logback.configuration: The org.eclipse.m2e.logback.configuration bundle was activated before the state location was initialized.  Will retry after the state location is initialized.
log4j:WARN No appenders could be found for logger (org.apache.http.client.protocol.RequestAddCookies).
log4j:WARN Please initialize the log4j system properly.
Installing org.eclipse.tm.terminal.feature.feature.group 4.5.100.201903081602.
Installing org.eclipse.tm.terminal.view.feature.feature.group 4.5.100.201903081602.
Installing org.eclipse.tm.terminal.control.feature.feature.group 4.5.100.201903081602.
Installing org.eclipse.tm.terminal.connector.ssh.feature.feature.group 4.5.100.201903081602.
Installing org.eclipse.tm.terminal.connector.telnet.feature.feature.group 4.5.100.201903081602.
Installation failed.
Cannot complete the install because one or more required items could not be found.
	Software being installed: TM Terminal 4.5.100.201903081602 (org.eclipse.tm.terminal.feature.feature.group 4.5.100.201903081602)
	Missing requirement for filter properties ~= $0: Terminal Local Connector 4.5.100.201901312304 (org.eclipse.tm.terminal.connector.local 4.5.100.201901312304) requires 'org.eclipse.equinox.p2.iu; org.eclipse.cdt.core.macosx 0.0.0' but it could not be found
	Cannot satisfy dependency:
		From: TM Terminal Local Connector Extensions 4.5.100.201903081602 (org.eclipse.tm.terminal.connector.local.feature.feature.group 4.5.100.201903081602)
		To: org.eclipse.equinox.p2.iu; org.eclipse.tm.terminal.connector.local [4.5.100.201901312304,4.5.100.201901312304]
	Cannot satisfy dependency:
		From: TM Terminal 4.5.100.201903081602 (org.eclipse.tm.terminal.feature.feature.group 4.5.100.201903081602)
		To: org.eclipse.equinox.p2.iu; org.eclipse.tm.terminal.connector.local.feature.feature.group 0.0.0
There were errors. See log file: /projects/software/default/eclipse/2019-06/Eclipse.app/Contents/Eclipse/configuration/1566404373251.log

******** ATTENTION ********
WARNING: Installation of plugin org.eclipse.tm.terminal.feature failed. Please try to install manually.
```

**Comment by dhelmke** [comment]
For me this line worked:

start eclipse -application org.eclipse.equinox.p2.director -repository http://download.eclipse.org/tm/terminal/marketplace -installIU org.eclipse.tm.terminal.feature.feature.group

it installed the TM terminal. But i dont know, how to integrate it in the devon ide. I am a newbie here.

**Comment by hohwille** [comment]
Awesome. Thanks. I will catch up from here...

**Comment by hohwille** [comment]
Fixed with PR #227
